FoodplannerApi/Program.cs

```diff
@@ -49,7 +49,7 @@
 app.MapControllers();
 
 app.MapGet("/test", () => {
-    var connectionString = SecretsLoader.GetSecret("DB_CONNECTION_STRING");
+    var connectionString = SecretsLoader.GetSecret("DB_CONNECTION_STRING", "/SW-5-02/");
```

Comment:
Kan man ikke bruge connectionFactory dingenoten her?

Reply:
Det er noget gammelt noget honestly som bare blev brugt til at tjekke om vi faktisk fik den secret vi havde brug for.